apps/backend/review/src/index.ts

```diff
@@ -1,25 +1,36 @@
 import express, { Request, Response, NextFunction } from "express";
 import morgan from "morgan";
-
-import { run } from "../../lib/src/example";
+import pgPromise from "pg-promise";
+import { getReview } from "./getReview";
+import { createReview } from "./createReview";
+import { updateReview } from "./updateReview";
+import { deleteReview } from "./deleteReview";
 
 const PORT = 8213;
 
 const app = express();
+
 app.use(morgan("dev"));
 app.use(express.json());
 
+const pgp = pgPromise();
+const DB_ENDPOINT = process.env.DB_ENDPOINT || "";
```

Comment:
This should error, in no scenario do we want it to try and connect to a database at empty string

Reply:
linter isn't happy with this. [nodemon] restarting due to changes...
[nodemon] starting `ts-node src/index.ts`
/app/service/node_modules/ts-node/src/index.ts:859
    return new TSError(diagnosticText, diagnosticCodes, diagnostics);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/index.ts(19,16): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string | IConnectionParameters<IClient>'.
  Type 'undefined' is not assignable to type 'string | IConnectionParameters<IClient>'.

    at createTSError (/app/service/node_modules/ts-node/src/index.ts:859:12)
    at reportTSError (/app/service/node_modules/ts-node/src/index.ts:863:19)
    at getOutput (/app/service/node_modules/ts-node/src/index.ts:1077:36)
    at Object.compile (/app/service/node_modules/ts-node/src/index.ts:1433:41)
    at Module.m._compile (/app/service/node_modules/ts-node/src/index.ts:1617:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Object.require.extensions.<computed> [as .ts] (/app/service/node_modules/ts-node/src/index.ts:1621:12)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12) {
  diagnosticCodes: [ 2345 ]
}
[nodemon] app crashed - waiting for file changes before starting...

Reply:
you can handle this error by adding a check below
`if(DB_ENDPOINT === "") {`
`    // handle error`
`}`

Reply:
```suggestion
const DB_ENDPOINT = process.env.DB_ENDPOINT;
if (!DB_ENDPOINT) {
	log + error here
}
```